### PR TITLE
Release 2023.01.2

### PR DESCRIPTION
**Pull request by isaacdurham-okta**
## Description:
- **What's changed?** Release Notes & footer text 
- **Is this PR related to a Monolith release?** 2023.01.2

### Resolves:

* [OKTA-568344](https://oktainc.atlassian.net/browse/OKTA-568344)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md [hunk 1]
```diff
@@ -6,11 +6,44 @@ title: Okta Identity Engine API Products release notes 2023
...
+| Change | Expected in Preview Orgs |
+|------------------------------------------------------------------------------------------------------------------|-----------------|
+| [Clock skew for access and ID tokens](#clock-skew-for-access-and-id-tokens)                                      |February 1, 2023 |
+| [Content Security Policy update](#content-security-policy-update)                                                |February 1, 2023 |
+| [Interaction Code flow supports Native SSO](#interaction-code-flow-supports-native-sso)                          |February 1, 2023 |
+| [Bugs fixed in 2023.01.2](#bugs-fixed-in-2023-01-2)                                                              |February 1, 2023 |
+
+#### Clock skew for access and ID tokens
+
+A 30 second clock skew is now allowed for access and ID tokens to validate that a token was created before the current time. <!--OKTA-538956 -->
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:20]:
```suggestion
A 30-second clock skew is now allowed for access and ID tokens to validate that a token was created before the current time. <!--OKTA-538956 -->
```

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md [hunk 2]
```diff
@@ -6,11 +6,44 @@ title: Okta Identity Engine API Products release notes 2023
...
+
+#### Content Security Policy update
+
+To improve our security and protect against Cross-Site Scripting (XSS) attacks, we are expanding the use of [Content Security Policy (CSP)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) to include when requests come from custom domains on non-customizable pages. <!--OKTA-567669 -->
+
+#### Interaction Code flow supports Native SSO
+
+The Interaction Code flow now supports the `device_sso` scope that you can use to perform Native SSO. This scope allows you to obtain an interaction code and exchange it for tokens and a device secret. See [Implement authorization by grant type](https://developer.okta.com/docs/guides/implement-grant-type/interactioncode/main/). <!--OKTA-551724 -->
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:28]: This edit was in the edit spreadsheet 
```suggestion
The Interaction Code flow now supports the `device_sso` scope, which you can use to perform Native SSO. This scope allows you to obtain an interaction code and exchange it for tokens and a device secret. See [Implement authorization by grant type](https://developer.okta.com/docs/guides/implement-grant-type/interactioncode/main/). <!--OKTA-551724 -->
```

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md [hunk 3]
```diff
@@ -6,11 +6,44 @@ title: Okta Identity Engine API Products release notes 2023
...
+
+#### Bugs fixed in 2023.01.2
+
+* Requests made with an empty Origin header returned an HTTP 500 error response. (OKTA-449621)
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:32]: edit was in the edit spreadsheet

```suggestion
* Requests made with an empty Origin header returned an error response. (OKTA-449621)
```

File: packages/@okta/vuepress-site/docs/release-notes/2023/index.md [hunk 4]
```diff
@@ -4,6 +4,36 @@ title: Okta API Products release notes 2023
...
+
+| Change | Expected in Preview Orgs |
+|------------------------------------------------------------------------------------------------------------------|-----------------|
+| [Clock skew for access and ID tokens](#clock-skew-for-access-and-id-tokens)                                      |February 1, 2023 |
+| [Content Security Policy update](#content-security-policy-update)                                                |February 1, 2023 |
+| [Bugs fixed in 2023.01.2](#bugs-fixed-in-2023-01-2)                                                              |February 1, 2023 |
+
+#### Clock skew for access and ID tokens
+
+A 30 second clock skew is now allowed for access and ID tokens to validate that a token was created before the current time. <!--OKTA-538956 -->
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023/index.md:17]:
```suggestion
A 30-second clock skew is now allowed for access and ID tokens to validate that a token was created before the current time. <!--OKTA-538956 -->
```

File: packages/@okta/vuepress-site/docs/release-notes/2023/index.md [hunk 5]
```diff
@@ -4,6 +4,36 @@ title: Okta API Products release notes 2023
...
+
+#### Content Security Policy update
+
+To improve our security and protect against Cross-Site Scripting (XSS) attacks, we are expanding the use of [Content Security Policy (CSP)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) to include when requests come from custom domains on non-customizable pages. <!--OKTA-567669 -->
+
+#### Bugs fixed in 2023.01.2
+
+* Requests made with an empty Origin header returned an HTTP 500 error response. (OKTA-449621)
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023/index.md:25]:
```suggestion
* Requests made with an empty Origin header returned an error response. (OKTA-449621)
```

File: packages/@okta/vuepress-site/docs/release-notes/2023/index.md [hunk 6]
```diff
@@ -4,6 +4,36 @@ title: Okta API Products release notes 2023
...
+
+* An error occurred if a request was made to the `/authorize` endpoint of a custom authorization server and the `prompt` parameter value was set to `enroll_authenticator`.  (OKTA-552063)
+
+* Requests to assign a custom role to a user or group returned a generic Bad Request error when the required `role` or `resource-set` property was missing. (OKTA-538237)
+
+* Some custom admins didn't see groups and users that they had access to when they made a `GET` request to the Users (`/users/{id}/groups`) and Groups (`/groups/{id}/users`) APIs. (OKTA-568838)
+
+* When a client used the private_key_jwt client authentication method, the `max_age` and `login_hint` parameters in the authorize request were sometimes ignored. (OKTA-534942)
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023/index.md:33]:
```suggestion
* When a client used the `private_key_jwt` client authentication method, the `max_age` and `login_hint` parameters in the authorize request were sometimes ignored. (OKTA-534942)
```

**Review by susanharper-okta — COMMENTED** [review]
couple comments

File: packages/@okta/vuepress-site/docs/release-notes/2023/index.md (hunk 4 above)
Comment (isaacdurham-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2023/index.md:17]: Adjusted.

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md (hunk 2 above)
Comment (isaacdurham-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:28]: Adjusted.

File: packages/@okta/vuepress-site/docs/release-notes/2023/index.md (hunk 5 above)
Comment (isaacdurham-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2023/index.md:25]: Adjusted.

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**